src/execution/operator/order/physical_top_n.cpp

```diff
@@ -260,6 +268,9 @@ void TopNHeap::Finalize() {
 }
 
 void TopNHeap::Reduce() {
+	if (limit == NumericLimits<idx_t>::Maximum()) {
```

Comment:
Please revert any changes made to the Top N operator. A Top-N operator that is optimized for a percentage could be created in a later PR but for now leave it as-is.

Reply:
Okay. I will revert any changes made to the Top N operator.
By the way, I have a question.
Then can't use at same time "order by" and "limit %" clause. isn't it?

Reply:
No, the Top N node is just an optimization. It is not required for the system to function.